src/views/virtualmachines/details/tabs/configuration/network/utils/utils.ts

```diff
@@ -65,3 +76,38 @@ export const isPendingRemoval = (
 
   return interfaceNotFound(vm, nicName) && isActiveOnGuest(vmi, nicName, isVMRunning);
 };
+
+export const getInterfaceState = (vm: V1VirtualMachine, nicName: string): NetworkInterfaceState => {
+  const simpleIfaceState = getNetworkInterfaceState(vm, nicName);
+  return isEmpty(simpleIfaceState)
+    ? NetworkInterfaceState.UP
```

Comment:
absent/unknown seems a better choice here

Reply:
There are three possible values for the `state` field: `up`, `down`, and `absent`. The `state` field is optional though, so we also have to account for it not being present.

I was told that if the `state` field isn't present and it's not an SR-IOV interface to show the link state as `up`.

I hope I'm addressing the right concern here. If not, let me know.

Reply:
OK. It's a feature:)